packages/utils/package.json

```diff
@@ -8,6 +8,14 @@
     ".": {
       "types": "./dist/index.d.ts",
       "import": "./dist/index.js"
+    },
+    "./libp2p": {
```

Comment:
You need to update the rollup config file with the same path

Reply:
You can discard this comment once https://github.com/waku-org/js-waku/pull/1204 is merged